### PR TITLE
Resprites gibber, food processor, grinder, bottler.

### PR DESCRIPTION
**Pull request by S34NW**
## What Does This PR Do
Ports grinder ("juicer") sprites from https://github.com/yogstation13/Yogstation/pull/9182
Adapted that to replace our bottler sprite with something slightly more 3/4ths.
Ports eris gibber and food processor, literally no idea who sprited these as there is no blame trail for them.

## Why It's Good For The Game
Old front-on sprites look out of place with all our updated visuals. The purge continues (I'm coming for you linen bin)

## Images of changes
image

https://user-images.githubusercontent.com/12197162/223277070-b5240525-9276-4c64-8548-e7770c113d97.mp4

## Testing
Gibbed my bald man

## Changelog
:cl: Partheo, S34N
tweak: Resprites gibber, food processor, grinder, bottler
/:cl:

### CODE REVIEW

**Review by Sirryan2002 — APPROVED** [review]

File: code/modules/food_and_drinks/kitchen_machinery/gibber.dm [hunk 1]
```diff
@@ -45,23 +45,30 @@
...
 	user.forceMove(src)
 	occupant = user
-	update_icon(UPDATE_OVERLAYS)
+	update_icon(UPDATE_OVERLAYS | UPDATE_ICON_STATE)
 	feedinTopanim()
 	addtimer(CALLBACK(src, PROC_REF(startgibbing), user), 33)
 	return OBLITERATION
 
+/obj/machinery/gibber/update_icon_state()
+	. = ..()
```
Comment (farie82) [inline code/modules/food_and_drinks/kitchen_machinery/gibber.dm:54]:
```suggestion
```
Has no use to call the parent here right?

**Review by farie82 — APPROVED** [review]
Minor thing. Looks good otherwise

File: code/modules/food_and_drinks/kitchen_machinery/gibber.dm (hunk 1 above)
Comment (S34NW) [reply code/modules/food_and_drinks/kitchen_machinery/gibber.dm:54]: Ah, no I was mixing it up with `update_icon`

**Review by farie82 — APPROVED** [review]

**Comment by farie82** [comment]
GitHub ain't letting me merge it. Closing and reopening to see if that fixes it

**Pull request merged**